bundles/org.openhab.core.karaf/src/main/java/org/openhab/core/karaf/internal/FeatureInstaller.java

```diff
@@ -56,19 +57,27 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * This service reads addons.cfg and installs listed addons (= Karaf features) and the selected package.
+ * This service reads addons.cfg and installs listed add-ons (= Karaf features) and the selected package.
  * It furthermore allows configuration of the base package through the Paper UI as well as administrating Karaf to
  * access remote repos and certain feature repos like for legacy or experimental features.
  *
  * @author Kai Kreuzer - Initial contribution
  */
 @Component(name = "org.openhab.addons", service = { FeatureInstaller.class, ConfigurationListener.class }, property = {
```

Comment:
One question: Do we need to define the `configurationPid` for this service?

Reply:
It seems to work without, so probably not.
If I am not mistaken, if there is no dedicated configurationPid, the service name property is used.

Reply:
Yes, that is right.